### DISCUSSION TITLE
Webhook [Jun 21]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2501] [Discord Threads]
> 🙋‍♂️ **Bergi9**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Bergi9**  
#### Is there a way to disable this param to force trigger the webhooks?  
#  
> 👨‍🦲 **dstala**  
#### <@269909921724629004> Noticed this behaviour. We are working on providing a fix for it; we should have it in our next release.  
#  
> 🙋‍♂️ **Bergi9**  
#### thank you  
#  
> 🙋‍♂️ **Bergi9**  
#### Expected release date?  
#  
> 👨‍🦲 **dstala**  
#### Hopefully by end of this week. Will keep you posted here.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/987316378686091325)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
